test/expected/pg_dump.out

```diff
@@ -41,7 +40,7 @@ SELECT count(*)
      AND refobjid = (SELECT oid FROM pg_extension WHERE extname = 'timescaledb');
  count 
 -------
-   206
+   100
```

Comment:
200ish -> 100, could that indicate that only one partition is dumped or is this expected ?